crates/pathfinder/src/rpc/types.rs

```diff
@@ -449,8 +449,7 @@ pub mod reply {
     /// Implements spec version [v0.1.0-rc1](https://github.com/starkware-libs/starknet-specs/releases/tag/v0.1.0-rc1)
     /// plus this PR: ["Pack storage diff entries per contract address"](https://github.com/starkware-libs/starknet-specs/pull/26)
     #[skip_serializing_none]
-    #[derive(Clone, Debug, Serialize, PartialEq)]
-    #[cfg_attr(any(test, feature = "rpc-full-serde"), derive(serde::Deserialize))]
+    #[derive(Clone, Debug, serde::Deserialize, Serialize, PartialEq)]
```

Comment:
I think it'd be worth a comment why this type doesn't follow the "implement Deserialize only for tests" schema other RPC types are using.

Reply:
As per you previous comment, depending on whether the rpc type stays in the DB I'll:
- remove the annotation if sequencer type is stored instead,
- add a proper comment otherwise.